app/views/admin/users/reports/_index.html.erb

```diff
@@ -4,23 +4,27 @@
   <div class="pt-3">
     <% @related_reports.each do |report| %>
       <article class="c-list-item c-list-item--interactive flex justify-between gap-4">
-        <div>
-          <header class="flex gap-3 items-center">
-            <h3 class="crayons-subtitle-3">
-              <a href="<%= admin_report_path(report.id) %>" class="c-link">
-                <%= report.category.capitalize %>
-              </a>
+        <div class="w-100">
+          <header>
+            <h3 class="crayons-subtitle-3 w-100 flex items-center justify-between">
```

Comment:
This should all look the same in the UI, I've just shuffled it around so that the heading is more descriptive for AT users. Previously all of the headings would have been categories  (e.g. many headings all saying "Spam"), which doesn't really give enough context. Now screen reader users will hear e.g. "Spam Open 11 Feb 22" which feels a little bit more useful at a glance.